AGGREGATE.md

```diff
@@ -183,7 +183,7 @@ trusted aggregation service known by the browser.
 
 The report will be JSON encoded with the following scheme:
 
-```
+```jsonc
 {
   "source_site": "https://publisher.example",
   "attribution_destination": "https://advertiser.example",
```

Comment:
Highlighting below is a bit funny as they are being highlighting as JSON rather than values.

But I think that is moot as both of these should be encoded as strings anyway as they represent 64 bit unsigned ints.

"source_registration_time": "[timestamp in seconds]"
“report_id”: "[64 bit unsigned integer]"

Reply:
Done